uat/custom-components/client-java-paho/src/main/java/com/aws/greengrass/testing/mqtt311/client/paho/Mqtt311ConnectionImpl.java

```diff
@@ -109,7 +114,7 @@ public MqttSubscribeReply subscribe(long timeout, @NonNull List<Subscription> su
             logger.atError().withThrowable(e).log("Exception occurred during subscribe, reason code {}",
                                                     e.getReasonCode());
 
-            builder.addAllReasonCodes(Collections.nCopies(subscriptions.size(), (int)e.getReasonCode()));
+            throw new RuntimeException(e.getMessage(), e);
```

Comment:
We need new description of throw here. Do not use the same message. 
Also method going to throws MqttException but here unckecked RuntimeException is thrown, why ?